### ISSUE TITLE
Container supabase-analytcs error / Self hosting with docker

### ISSUE THREAD

**Issue by GillesClerc** [supabase/supabase#30232]
# Bug report

- [X] I confirm this is a bug with Supabase, not with my own application. 
- [X] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

Container supabase-analytics doesn't start anymore after changing the password as requested on the documentation: https://supabase.com/docs/guides/self-hosting/docker

## To Reproduce

1. Change default POSTGRES_PASSWORD on .env with new one 
2. Restart docker with "docker compose down" and "docker compose up -d"

## Expected behavior

The container Supabase-analytics to run

## System information

- OS: Ubuntu 24.04
- Self hosted with docker

**Comment by tsensei** [comment]
Easy answer : Before getting it up after changing password, you'll have to delete the /db/data volume directory, that will also delete all data, so have a backup

Refer to this issue for better understanding : https://github.com/supabase/supabase/issues/22605

**Comment by modyabhi** [comment]
> Easy answer : Before getting it up after changing password, you'll have to delete the /db/data volume directory, that will also delete all data, so have a backup
> 
> Refer to this issue for better understanding : #22605

Ok this works by `rm -rf volumes/db/data/`  but this solution kind of sucks. Having to delete your entire data may not be viable for people who are already in prod. 

This is a genuine bug.

**Comment by tsensei** [comment]
Agreed, the mentioned thread has better ways or you could use the CLI to
dump the schema and data separately the just apply those in the new DB and
it should be in the same state.

**Comment by AvantaSolutions** [comment]
I updated the mentioned thread with step by step instructions on how I was able to do this without removing volumes/db/data in case you're still looking for a solution to this.

**Comment by encima** [comment]
Hello all,

Please note that you can change the password for the database using standard means.
`ALTER USER postgres WITH PASSWORD 'pwd';`

We can update the docs to address this!

**Comment by Hallidayo** [comment]
Hi all, We’re going to close this one due to inactivity. Please feel free to reopen if you’re still having issues here or open a new issue.
